src/components/auth/Login.js

```diff
@@ -24,12 +25,24 @@ function LoginForm({ authLogin, loading, error, errResponse, token, history }) {
       : (ref.current.firstChild.type = 'password');
   };
 
+  useEffect(() => {
+    if (localStorage.getItem('codeclan_token')) {
+      history.push('/dashboard');
+    }
+  }, []);
```

Comment:
Codacy found an issue: [React Hook useEffect has a missing dependency: 'resetAuthState'. Either include it or remove the dependency array.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=6042181)